app/(afterLogin)/(deal)/deals/activity/page.tsx

```
@@ -0,0 +1,320 @@
+<div className="h-8 w-8 rounded-full bg-sky-50 flex items-center justify-center ring-1 ring-sky-200">
+    <Icon className="text-sky-500" />
+  </div>
+            </div>
+            <div className="mt-1">
+              <PriorityTag level={priority} />
+              <div className="font-semibold text-slate-800">{title}</div>
+              <PersonMeta name={person} deal={deal} />
+            </div>
+            <div className="flex justify-between items-center gap-2">
+              <div className="flex-1">
+                  <div className="mt-3">
```

ESLint

[error] Delete `··`

(prettier/prettier)